src/app_engine/apprtc.py

```diff
@@ -119,7 +119,22 @@ def get_wss_parameters(request):
   ws_tls = request.get('wstls')
 
   if not ws_host_port_pair:
-    ws_host_port_pair = constants.WSS_HOST_PORT_PAIR
+    memcache_client = memcache.Client()
+    wss_host_status_list = memcache_client.get(
+        constants.WSS_HOST_STATUS_LIST_KEY)
+    # Return the first server in the list that is up, which is the wss host
+    # that has been up the longest.
+    if wss_host_status_list is not None:
+      wss_host_status_list = [host for host in wss_host_status_list
```

Comment:
Another way to do this which I think is a bit more Pythonic is,

```python
try:
  ws_host = (host for host in wss_host_status_list if host[constants.WSS_HOST_IS_UP_KEY]).next()
  ws_host_port_pair = ws_host[constants.WSS_HOST_INSTANCE_KEY]
except StopIteration:
  logging.warning('no wss servers are reported as up, using fallback')
  ws_host_port_pair = constants.WSS_HOST_PORT_PAIRS[0]
```

My arguments:

1. using a try is better in cases where you assume it will succeed and I think in *general* we assume this will succeed; i.e., one host should be up.
2. It avoids iteration through the entire list of hosts. Doesn't matter with our small list I guess.